test/e2e/utils_test.go

```diff
@@ -0,0 +1,151 @@
+package e2e_test
+
+import (
+	"fmt"
+	"math/rand"
+	"os/exec"
+	"testing"
+	"time"
+
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	"github.com/openshift/machine-config-operator/test/e2e/framework"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
```

Comment:
you need to import corev1 or this will keep failing:

corev1 "k8s.io/api/core/v1"

Reply:
also do you need "k8s.io/apimachinery/pkg/labels"?

Reply:
that's what you get when you don't run the tests first :sweat_smile: